src/phoenix/server/api/dataloaders/span_by_id.py

```diff
@@ -0,0 +1,26 @@
+from typing import Union
+
+from sqlalchemy import select
+from strawberry.dataloader import DataLoader
+from typing_extensions import TypeAlias
+
+from phoenix.db import models
+from phoenix.server.types import DbSessionFactory
+
+Key: TypeAlias = int
+Result: TypeAlias = models.Span
+
+
+class SpanByIdDataLoader(DataLoader[Key, Result]):
+    def __init__(self, db: DbSessionFactory) -> None:
+        super().__init__(load_fn=self._load_fn)
+        self._db = db
+
+    async def _load_fn(self, keys: list[Key]) -> list[Union[Result, ValueError]]:
+        ids = list(set(keys))
+        spans: dict[Key, Result] = {}
+        async with self._db() as session:
+            data = await session.stream_scalars(select(models.Span).where(models.Span.id.in_(ids)))
```

Comment:
so this models.Span doesn't pull in the entire attributes block is that correct?